src/Command/Api4Command.php

```diff
@@ -149,11 +149,14 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       $columns = empty($params['select']) ? array_keys($result->first()) : $params['select'];
       $this->sendTable($input, $output, (array) $result, $columns);
     }
+    elseif (!empty($params['select']) && $params['select'] === ['row_count']) {
+      $this->sendResult($input, $output, $result->count());
+    }
     else {
       $this->sendResult($input, $output, $result);
     }
 
-    return empty($result['is_error']) ? 0 : 1;
+    return 0;
```

Comment:
I don't see how this part is relevant. Note that the result-code is important for handling errors in bash scripts. Returning `0` signals normal/successful exit. Errors should return non-zero. (*Some tools go all-out and define various numeric-error-codes for different conditions. That's a bit of PITA here, which is why it merely returns `1`.*)

Some examples:

```bash
set -e
cd path/to/my/ext
git pull
cv upgrade:db
cd ..
```

```bash
set -e
cv api4 Contact.get +s display_name --out=list | while read NAME; do
   echo "Say hello to $NAME for me" 
done
```

```bash
cv flush && cv ext:list -R
```

In the first two cases, `set -e` means "If any command fails, stop execution and report the problem". If the third case, the `&&` means "If left-command succeeds, then run right-command." They all rely on the exit-code.

Reply:
It's just cleanup. This function looks to have been copy/pasted from the api3 function. Api3 returns an `is_error` key. Apiv4 does not.